### PR TITLE
Whole InfoCard clickable when only one link

### PR DESCRIPTION
**Pull request by sennett**
# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

**Comment by paszkowskiDamian** [comment]
image

The target of anchor is not covering entire card

File: components/InfoCard.tsx [hunk 1]
```diff
@@ -1,54 +1,131 @@
...
     <Card
       sx={{
         ...sx,
-        p: 4,
         borderRadius: 'large',
         border: 'none',
         backgroundImage: `url(${staticFilesRuntimeUrl(backgroundImage)}), ${backgroundGradient}`,
         backgroundPosition: 'bottom 0px right 0px',
         backgroundRepeat: 'no-repeat',
+        // backgroundSize: '50%',
```
Comment (piekczyk) [inline components/InfoCard.tsx]: to be removed

File: components/InfoCard.tsx [hunk 2]
```diff
@@ -1,54 +1,131 @@
...
+  title: string
+  subtitle: string
+  links: Array<{ href: string; text: string }>
+  backgroundImage: string
+  backgroundGradient: string
+  sx?: SxStyleProp
+}
+
+export function InfoCard(props: InfoCardProps) {
+  if (props.links.length === 1) {
```
Comment (piekczyk) [inline components/InfoCard.tsx:66]: I guess you could use here ternary operator instead of if / else.

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: components/InfoCard.tsx (hunk 2 above)
Comment (paszkowskiDamian) [reply components/InfoCard.tsx:66]: I would leave `if` but remove else. 

```
if (something) {
return </>
}

return </>
```

3 developers 3 opinions :)

**Review by paszkowskiDamian — APPROVED** [review]

**Review by paszkowskiDamian — APPROVED** [review]

File: components/InfoCard.tsx (hunk 1 above)
Comment (sennett) [reply components/InfoCard.tsx]: Done

File: components/InfoCard.tsx (hunk 2 above)
Comment (sennett) [reply components/InfoCard.tsx:66]: ¯\_(ツ)_/¯

**Review by piekczyk — APPROVED** [review]

**Pull request merged**